join.html

```diff
@@ -82,9 +89,9 @@
     </button>
     <nav>
       <div class="hamburger">
-        <span class="bar" />
-        <span class="bar" />
-        <span class="bar" />
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
```

Comment:
Why this change is made?

Reply:
self-closing span broke the join section. you can check it out at realdevsquad.com/join.html

Reply:
I can see that self closing span tags are actually being nested in DOM 
![image](https://user-images.githubusercontent.com/78433013/197000426-7af9a3de-42e3-4c1e-85d5-20ef33b6cef8.png)

Reply:
Interesting to debug, why is that so